_redirects

```diff
@@ -170,6 +170,7 @@
 /docs/setting-up-enterprise-sso-with-azure-active-directory	/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory	302
 /docs/setting-up-snowflake-sso	/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-snowflake-oauth	302
 /docs/setting-up-sso-with-google-gsuite	/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite	302
+/img/docs/dbt-cloud/dbt-cloud-enterprise/icon.png	https://www.getdbt.com/ui/img/dbt-icon.png	302
```

Comment:
It looks like this redirect isn't firing, as it still loads the image from docs: [img preview](https://deploy-preview-1271--docs-getdbt-com.netlify.app/img/docs/dbt-cloud/dbt-cloud-enterprise/icon.png)

I _think_ the 302 will need an exclamation point after it, so `302!`. Per [Netlify article](https://docs.netlify.com/routing/redirects/rewrites-proxies/): `If you’re 100% sure that you’ll always want to redirect, even when the URL matches a static file, you can append an exclamation mark to the rule:`

I don't think it would hurt to change `302` to `301` here either, as 301 = permanent redirect, 302 = temporary redirect. Setting to 301 should help google know to always go to the new url.